src/content/en/updates/2017/02/media-session.md

```diff
@@ -0,0 +1,398 @@
+
+## Get into the code
+
+### Let's play 🎷
+
+Add a simple `<audio>` tag to your web page and assign several media sources so
+that the browser can choose which one works best.
+
+    <audio controls>
+      <source src="audio.mp3" type="audio/mp3"/>
+      <source src="audio.ogg" type="audio/ogg"/>
+    </audio>
+
+Note: I could have used a `<video>` tag as well there to showcase the Media
+Session API.
+
+As you may know, `autoplay` is disabled on Chrome for Android which
+means we have to use the `play()` method of the audio element there. This
+method must be triggered by [a user gesture] such as a touch or a mouse click.
+We're talking about listening to
+[`pointerup`](/web/updates/2016/10/pointer-events), `click`, and `touchend`
+events. In other words, the user must click on a button before your web app can
+actually make some noise.
+
+    playButton.addEventListener('pointerup', function(event) {
+      let audio = document.querySelector('audio');
+
+      // User interacted with the page. Let's play audio...
+      audio.play()
+      .then(_ => { /* Set up Media Session... */ })
+      .catch(error => { console.log(error) });
+    });
+
+If you don't want to play audio right after the first interaction, I recommend
+you use the `load()` method of the audio element. This is one way for the
+browser to keep track of whether the user interacted with the element. Note
+that it may also help the playback to be smoother because the content will be
+loaded.
+
+<pre class="prettyprint">
+
+let audio = document.querySelector('audio');
+
+welcomeButton.addEventListener('pointerup', function(event) {
+  // User interacted with the page. Let's load audio...
+  <strong>audio.load()</strong>
+  .then(_ => { /* Show play button for instance... */ })
+  .catch(error => { console.log(error) });
+});
+
+// Later...
+playButton.addEventListener('pointerup', function(event) {
+  <strong>audio.play()</strong>
+  .then(_ => { /* Set up Media Session... */ })
+  .catch(error => { console.log(error) });
+});
+
+</pre>
+
+### Customize the notification
+
+When your web app is playing audio, you can already see a media notification
+sitting in the notification tray. On Android, Chrome does its best there to show
+appropriate information by using the document's title and the largest favicon
+it can find.
+
+<div class="clearfix"></div>
+<div class="attempt-left">
+  <figure>
+    <img src="/web/updates/images/2017/02/without-media-session.png" alt="Without Media Session">
+    <figcaption>With no Media Session</figcaption>
+  </figure>
+</div>
+<div class="attempt-right">
+  <figure>
+    <img src="/web/updates/images/2017/02/with-media-session.png" alt="With Media Session">
+    <figcaption>With Media Session</figcaption>
+  </figure>
+</div>
+
+Let's see how to customize this media notification by setting some media
+session metadata such as the title, artist, album name, and artwork with the
+Media Session API.
+
+    if ('mediaSession' in navigator) {
```

Comment:
It's kinda covered in earlier code, but is it worth mentioning when this code should execute.

Maybe just a comment like:

```js
// When the audio starts playing…
```